google-cloud-dialogflow/MIGRATING.md

```diff
@@ -285,6 +290,94 @@ def foo
 end
 ```
 
+### Streaming Interface
+
+The client library includes one special streaming method `streaming_detect_intent`.
+In the older client, this method provided only a very basic Enumerable-based
+interface, and required you to write wrappers if you wanted more flexibility.
+In version 1.0, we have standardized the streaming interfaces across the various
+Ruby client libraries. The `streaming_detect_intent` call takes an input stream
+object that can be written to incrementally, and returns a lazy enumerable that
+you can query for incremental results.
+
+Old:
+```
+client = Google::Cloud::Dialogflow::Sessions.new
+
+# Build requests
+session = "projects/my-project/agent/sessions/my-session"
+header = {
+  session: session,
+  query_input: {
+    audio_config: {
+      audio_encoding: Google::Cloud:Dialogflow::V2::AudioEncoding::AUDIO_ENCODING_FLAC,
+      sample_rate_hertz: 44_000,
+      language_code: "en-US
+    }
+  }
+}
+data1 = {
+  session: session,
+  input_audio: File.read("data1.flac", mode: "rb")
+}
+data2 = {
+  session: session,
+  input_audio: File.read("data2.flac", mode: "rb")
+}
+
+# Issue the call
+responses = client.streaming_detect_intent [header, data1, data2]
+
+# Handle responses as they arrive
+responses.each do |response|
+  puts "received: #{response}"
+end
+```
+
+New:
+```
+client = Google::Cloud::Dialogflow.sessions
+
+# Create a request stream, initiate the call, and get a response stream.
+request_stream = Gapic::StreamInput.new
+response_stream = client.streaming_detect_response request_stream
```

Comment:
Shouldn't this be `client. streaming_detect_intent`?

Reply:
Thanks for catching that. Fixed.